app/views/schools/advice/electricity_long_term/_current_usage.html.erb

```diff
@@ -24,6 +24,7 @@
     </tr>
   </tbody>
 </table>
+<%= render 'schools/advice/how_have_we_analysed_your_data_table_caption', data_target: "how-have-we-analysed-your-data-footnotes" %>
```

Comment:
This won't look right with 2 separate lines. We need to combine into a single table footnote. Suggest just replacing the existing footnote with the new standard on, and moving the "Review our analysis" to a separate line without the right alignment